testgrid/cmd/configurator/prow.go

```diff
@@ -88,6 +90,12 @@ func applySingleProwjobAnnotations(c *Config, pc *prowConfig.Config, j prowConfi
 				TestGroupName: testGroupName,
 				Description:   description,
 			}
+			if firstDashboard {
+				firstDashboard = false
+				if emails, ok := j.Annotations[testgridEmailAnnotation]; ok {
+					dt.AlertOptions = &config.DashboardTabAlertOptions{AlertMailToAddresses: emails}
```

Comment:
I guess this means the alert e-mail is only setup for whatever dashboard_tab happens to be first in the annotations, can we document this somewhere?

Reply:
https://github.com/kubernetes/test-infra/blob/master/testgrid/README.md

Seems to be the right section.